sites/docs/docs/features/queries/number-formatting.md

```diff
@@ -123,7 +113,7 @@ When creating a table, Evidence formats column titles based on the name of the c
 </tr>
 <tr>
 <td>growth_pct</td>
-<td>Growth (%)</td>
```

Comment:
There was some inconsistency with the docs. Before the current formatting changes we were not showing % for instance.